staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -106,7 +106,7 @@ var (
 		&compbasemetrics.HistogramOpts{
 			Subsystem: APIServerComponent,
 			Name:      "request_slo_duration_seconds",
-			Help:      "Response latency distribution (not counting webhook duration) in seconds for each verb, group, version, resource, subresource, scope and component.",
+			Help:      "Response latency distribution (not counting webhook and priortiy & fairness duration) in seconds for each verb, group, version, resource, subresource, scope and component.",
```

Comment:
let's also add this change to `request_sli_duration_seconds` since `request_slo_duration_seconds` is going to be deprecated this release and hidden/removed in 1.28

Reply:
Done, thanks